packages/models/src/index.ts

```diff
@@ -132,3 +132,7 @@ export {
   type Tree,
 } from './lib/tree.js';
 export { uploadConfigSchema, type UploadConfig } from './lib/upload-config.js';
+export {
+  artifactGenerationCommand as toolCommandSchema,
```

Comment:
we have renamed `toolCommandSchema` to `artifactGenerationCommand`. Do not use it anymore even if it is a reexport.